yarn.lock

```diff
@@ -7613,6 +7632,11 @@ mkdirp@^0.5.1, mkdirp@^0.5.5:
   dependencies:
     minimist "^1.2.6"
 
+mobx@6.8.0:
```

Comment:
do you know where this dependency comes from? we probably don't want this in telescope unless it's dev dep